src/components/review/ProposalTableReviewer.tsx

```diff
@@ -49,6 +53,13 @@ const getFilterStatus = (selected: string | ReviewStatus) =>
     ? ReviewStatus.DRAFT
     : undefined; // if the selected status is not a valid status assume we want to see everything
 
+const getFilterReviewer = (selected: string | ReviewerFilter) =>
```

Comment:
This is a little bit hard to read, maybe this logic can be simplified?

Reply:
I just used the same logic as the previous `getFilterStatus` function but I agree maybe we should simplify both of them. I will try to do it.